internal/common/resource.go

```diff
@@ -490,6 +491,9 @@ func defaultUpdateFunc(newObj, foundObj client.Object) {
 	case *promv1.ServiceMonitor:
 		foundObj.(*promv1.ServiceMonitor).Spec = newTyped.Spec
 
+	case *v1beta1.Task:
```

Comment:
**future PR** - code style - please use a different name for the import. `v1beta1.Task` is unclear. `tekton.Task` would be more clear.

Reply:
Done.